pkg/report/minio/file.go

```diff
@@ -22,7 +22,7 @@ func (f *memoryFile) Stat() (os.FileInfo, error) {
 	return &memoryFileInfo{f}, nil
 }
 
-func (f *memoryFile) Readdir(count int) ([]os.FileInfo, error) {
+func (f *memoryFile) Readdir(_ int) ([]os.FileInfo, error) {
```

Comment:
Can be simplified even more

```suggestion
func (f *memoryFile) Readdir(int) ([]os.FileInfo, error) {
```